src/components/Switzerland/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useState, useRef } from 'react';
+import { DistributionType, getVariantDistributionData } from '../../services/api';
+import { TimeZipCodeDistributionEntry } from '../../services/api-types';
+import { AccountService } from '../../services/AccountService';
+import * as zod from 'zod';
+import { SampleSelectorSchema } from '../../helpers/sample-selector';
+import { useResizeDetector } from 'react-resize-detector';
+
+import Map from './Map';
+
+const PropsSchema = SampleSelectorSchema;
+type Props = zod.infer<typeof PropsSchema> & { width?: number };
```

Comment:
`& { width?: number };` is not used anymore.

Reply:
- [ ] Remove